glossary.rst

```diff
@@ -80,13 +80,13 @@ Glossary
 
   widget
 
-    A single question, answer set, and attendant GUI elements, as rendered in an XForm compliant app such as :term:`Collect`.
+    A single question, answer set, and attendant GUI elements, as rendered in a XForm compliant app such as :term:`Collect`.
 
   hint
 
     Additional help text on a single question, displayed after the label.
 
   question label
   question text
-  
-    The main part body of a form question or widget. In an :term:`XLSForm`, the contents of :th:`label`.
+
+    The main part body of a form question or widget. In a :term:`XLSForm`, the contents of the :th:`label`.
```

Comment:
"XForm" is pronounced *eks-form*, so "an XForm" is correct.

Reply:
Thanks for pointing it out, I was so focused on the grammatical aspect that I oversaw the phonetical part.